app/serializers/api/v2/serializable_course.rb

```diff
@@ -8,7 +8,7 @@ def enrichment_attribute(name, enrichment_name = name)
           end
         end
       end
-
+      include Courses::EditCourseOptions
```

Comment:
This should not be required when you've made the changes.

Reply:
Done 👍 thanks.

Reply:
Done. Thanks